src/components/Editor/DrawingBoard/Canvas/Worker/EraserWorker.ts

```diff
@@ -15,6 +15,8 @@ class EraserWorker implements Worker {
 
   private createID?: TimeTicket;
 
+  private size: number = 24;
```

Comment:
It would be nice to express that this variable is the size of the eraser rectangle.
```suggestion
  private eraserBoxSize: number = 24;
```